common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -171,6 +226,10 @@ case class LogMarkerToken(component: String, action: String, state: String) {
 
     def asFinish = copy(state = LoggingMarkers.finish)
     def asError = copy(state = LoggingMarkers.error)
+
+    def getServiceName(): String = {
+        component + "_" + action
```

Comment:
Here service name is name of logical unit within context of action execution. Ex. invoker_activationinit or invoker_dockerpause etc. So combination of two is span name here.